Failed to resolve entry for package

Hi,

**Describe the bug**

I encountered this error today

----

`[ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json.

    node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40970:10:
      40970 │     throw new Error(`Failed to resolve entry for package "${id}". ` +
            ╵           ^

    at packageEntryFailure ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40970:11)
    at resolvePackageEntry ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40966:9)
    at tryNodeResolve ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40773:20)
    at Context.resolveId ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40581:28)
    at Object.resolveId ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39254:55)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async resolve ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39466:26)
    at async [...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39626:34
    at async callback ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:933:28)
    at async handleRequest ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:713:30)

  This error came from the "onResolve" callback registered here:

    node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39616:18:
      39616 │             build.onResolve({
            ╵                   ~~~~~~~~~

    at setup ([...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39616:19)
    at handlePlugins ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:855:23)
    at Object.buildOrServe ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1149:7)
    at [...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:2110:17
    at new Promise (<anonymous>)
    at Object.build ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:2109:14)
    at Object.build ([...]/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1956:51)
    at [...]/node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:39414:54
    at Array.map (<anonymous>)

  The plugin "vite:dep-scan" was triggered by this import

    script:[...]/src/App.vue?id=0:74:7:
      74 │ import "@synaps-io/vue3-verify"
         ╵        ~~~~~~~~~~~~~~~~~~~~~~~~

Build failed with 1 error:
node_modules/.pnpm/vite@2.9.15_sass@1.54.5/node_modules/vite/dist/node/chunks/dep-689425f3.js:40970:10: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json.
Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@synaps-io/vue3-verify". The package may have incorrect main/module/exports specified in its package.json.
`

---

**Reproduce the bug**

using this template: https://github.com/antfu/vitesse
os: macbook pro m1 apple chip
package manager: pnpm

--- a/package.json
+++ b/package.json
@@ -1,12 +1,11 @@
 {
   "name": "@synaps-io/vue3-verify",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Synaps Vue SDK",
   "author": "Synaps",
   "license": "Apache 2.0",
   "repository": "synaps-io/vue3-verify",
   "main": "dist/vue3-verify.umd.js",
-  "module": "dist/vue3-verify.esm.js",
   "unpkg": "dist/vue3-verify.umd.min.js",
   "browser": {
     "./sfc": "src/vue3-verify.vue"

